src/openrct2/world/Map.cpp

```diff
@@ -2178,10 +2191,10 @@ TileElement* map_get_track_element_at_of_type(int32_t x, int32_t y, int32_t z, i
 TileElement* map_get_track_element_at_of_type_seq(int32_t x, int32_t y, int32_t z, int32_t trackType, int32_t sequence)
 {
     TileElement* tileElement = map_get_first_element_at(x >> 5, y >> 5);
+    if (tileElement == nullptr)
```

Comment:
revert. Previous code works fine and tends to be the standard for `map_get...`

Reply:
Same question as above, wouldn't it lead to a crash if `titleElement` is null?

Reply:
As above. It never gets to that bit of the loop due to the break